apm-agent-plugins/apm-servlet-plugin/src/test/java/co/elastic/apm/agent/servlet/ApmFilterTest.java

```diff
@@ -79,9 +79,10 @@ void testEndsTransaction() throws IOException, ServletException {
 
     @Test
     void testDisabled() throws IOException, ServletException {
-        when(tracer.getConfig(CoreConfiguration.class).isActive()).thenReturn(false);
+        tracer.pause();
         filterChain.doFilter(new MockHttpServletRequest(), new MockHttpServletResponse());
         assertThat(reporter.getTransactions()).hasSize(0);
+        tracer.resume();
```

Comment:
The state should be reset after each test. If there's an exception in this methods, the state should not leak into other tests. Probably best to call `tracer.resume()` in `co.elastic.apm.agent.AbstractInstrumentationTest#afterAll`

Reply:
Add to be done `@AfterEach`